platform/windows/detect.py

```diff
@@ -261,14 +262,31 @@ def build_res_file(target, source, env: "SConsEnvironment"):
         "arm32": "armv7-w64-mingw32",
         "arm64": "aarch64-w64-mingw32",
     }
-    cmdbase = "windres --include-dir . --target=" + arch_aliases[env["arch"]]
 
-    mingw_bin_prefix = get_mingw_bin_prefix(env["mingw_prefix"], env["arch"])
+    if env["platform_tools"]:
+        if env["use_windres"]:
+            cmdbase = "windres"
+            mingw_bin_prefix = get_mingw_bin_prefix(env["mingw_prefix"], env["arch"])
+        else:
+            cmdbase = "rc"
+            mingw_bin_prefix = ""
+    else:
+        cmdbase = env["RC"]
+        mingw_bin_prefix = ""
+
+    if env["use_windres"]:
+        cmdbase += " --include-dir . --target=" + arch_aliases[env["arch"]]
+    else:
+        # rc doesn't seem to have a target architecture arg.  So not passing.
+        cmdbase += " /i ."
```

Comment:
```suggestion
        cmdbase += " /nologo /i ."
```
To match default SCons behavior. AFAIK, there's no architecture flag.

Reply:
Ah this was my bad.  Accidentally left a comment in from a different build file I was using.  Will remove this.  Good catch! :)

Reply:
Oh wait.  This was not a mistake.  This was intentional.  This is matching the behavior Godot's detect.py currently uses when building via windres.  Removing that ``--target=[arch_here]`` flag would change the current build process.  I would recommend doing that in a different PR and keeping this PR solely for adding new functionality instead of changing any existing functionality so as to keep things organized.

Here is the line as it is now: https://github.com/godotengine/godot/blob/5708a3a02e00061e03366f2dabf8942df66fedca/platform/windows/detect.py#L264

Reply:
Ah I see what you were requesting.  Completed and done :)